### PR TITLE
feat(atom): send transaction script

### PR DESCRIPTION
**Pull request by mukeshsp**
Ticket: BG-67328

### CODE REVIEW

File: examples/js/atom/send-wallet-transaction.js
[the remark this answers is not in the document]
Comment (mukeshsp) [reply examples/js/atom/send-wallet-transaction.js:1]: yes

**Review by OttoAllmendinger — COMMENTED** [review]
flush

File: examples/js/atom/send-wallet-transaction.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const signedTransaction = await wallet.signTransaction({
+    txPrebuild: unsignedTx,
+    keychain: keychains[0],
+    walletPassphrase: walletPassphrase,
+    pubs: keychains.map((k) => k.pub),
+    reqId: unsignedTx.txRequestId,
+  });
+
+  // submit tx
+  const submittedTx = await bitgo
```
Comment (dpkjnr) [inline examples/js/atom/send-wallet-transaction.js:72]: I think this is not required for api version full

Comment (mukeshsp) [reply examples/js/atom/send-wallet-transaction.js]: removed !!

**Review by dpkjnr — APPROVED** [review]

**Review by dpkjnr — APPROVED** [review]

**Pull request merged**